src/main/java/org/graylog2/gelfclient/GelfConfiguration.java

```diff
@@ -147,6 +151,66 @@ public GelfConfiguration queueSize(final int size) {
     }
 
     /**
+     * Check if TLS option for the transport is enabled.
+     *
+     * @return true if TLS is enabled, false if disabled
+     */
+    public boolean isTls() {
+        return tls;
+    }
+
+    /**
+     * Enable/disable TLS for transport.
+     *
+     * @param tls {@code true} to enable TLS, {@code false} to disable it
+     * @return {@code this} instance
+     */
+    public GelfConfiguration tls(final boolean tls) {
```

Comment:
Maybe methods like `enableTls` and `disableTls` would fit better in the builder-like usage of this class.